components/dropdown/dropdown-opener-mixin.js

```diff
@@ -133,6 +267,68 @@ export const DropdownOpenerMixin = superclass => class extends superclass {
 		}
 		opener.setAttribute('aria-expanded', 'true');
 		opener.setAttribute('active', 'true');
+		this._isFading = false;
+	}
+
+	/* used by open-on-hover option */
+	async __onOpenerMouseEnter() {
+		// do not respond to hover events on mobile screens
+		const dropdownContent = this.__getContentElement();
+		if (dropdownContent._useMobileStyling) return;
+		clearTimeout(this._dismissTimerId);
+		if (!this._isOpen) await this.openDropdown(false);
+		this._closeTimerStop();
+		this._isHovering = true;
+	}
+
+	/* used by open-on-hover option */
+	async __onOpenerMouseLeave() {
+		// do not respond to hover events on mobile screens
+		const dropdownContent = this.__getContentElement();
+		if (dropdownContent._useMobileStyling) return;
+		this._isHovering = false;
+		if (this._isOpenedViaClick) return;
+		//Wait before closing so we don't lose hover when we jump from opener to card
+		clearTimeout(this._dismissTimerId);
+		await this.closeDropdown(true);
+	}
+
+	/* used by open-on-hover option */
+	__onOpenerTouch(e) {
+		//Prevents touch from triggering mouseover/hover behavior
+		e.preventDefault();
+		this._closeTimerStop();
+		if (this._isOpen) this.closeDropdown();
+		this._isOpenedViaClick = true;
+		if (!this._isOpen) this.openDropdown(true);
 	}
 
+	/* used by open-on-hover option */
+	_closeTimerStart() {
+		if (this._isOpen) return;
+		clearTimeout(this._setTimeoutId);
+		this._isFading = true;
+		this._setTimeoutId = setTimeout(() => {
+			this.closeDropdown(false);
+			this._isFading = false;
+		}, 400);
+	}
+
+	/* used by open-on-hover option */
+	_closeTimerStop() {
+		clearTimeout(this._setTimeoutId);
+		this._isFading = false;
+	}
+
+	/* used by open-on-hover option */
+	_onOutsideClick(e) {
+		if (!this._isOpen) return;
+		const isWithinDropdown = isComposedAncestor(this.__getContentElement(), (e.path || e.composedPath())[0]);
```

Comment:
I think I need a reminder - under what circumstances do we not have a `path`?  Normally I end up always using `composedPath` because I am interested in the complete path, but when I inspect these two in the console, they appear identical.  Mozilla only lists `composedPath` in their docs.

Reply:
Ah, looks like this was a copy-paste from the original user-profile-card code. My best guess is that the path || composedPath check was added before full browser support of composedPath. Looking at our other core code, looks like we are safe to just use composedPath. I'll make that change.